packages/desktop-client/src/components/accounts/MobileAccounts.js

```diff
@@ -0,0 +1,358 @@
+        type="financial"
+      />
+    </View>
+  );
+}
+
+export function AccountCard({ account, updated, getBalanceQuery, onSelect }) {
+  return (
+    <View
+      style={{
+        flex: '1 0 auto',
+        flexDirection: 'row',
+        backgroundColor: 'white',
+        boxShadow: `0 1px 1px ${colors.n7}`,
+        borderRadius: 6,
+        marginTop: 10
+      }}
+    >
+      <Button
+        onMouseDown={() => onSelect(account.id)}
+        style={{
+          flexDirection: 'row',
+          flex: 1,
+          alignItems: 'center',
+          borderRadius: 6,
+          '&:active': {
+            opacity: 0.1
+          }
+        }}
+      >
+        <View
+          style={{
+            flex: '1 auto',
+            height: 52,
+            marginTop: 10
+          }}
+        >
+          <View
+            style={{
+              flexDirection: 'row',
+              alignItems: 'center'
+            }}
+          >
+            <TextOneLine
+              style={[
+                styles.text,
+                {
+                  fontSize: 17,
+                  fontWeight: 600,
+                  color: updated ? colors.b2 : colors.n2,
+                  paddingRight: 30
+                }
+              ]}
+            >
+              {account.name}
+            </TextOneLine>
+            {account.bankId && (
+              <View
+                style={{
+                  backgroundColor: colors.g5,
+                  marginLeft: '-23px',
+                  width: 8,
+                  height: 8,
+                  borderRadius: 8
+                }}
+              />
+            )}
+          </View>
+          <View
+            style={{
+              flexDirection: 'row',
+              alignItems: 'center',
+              marginTop: '4px'
+            }}
+          >
+            <Text style={[styles.smallText, { color: colors.n5 }]}>
+              {prettyAccountType(account.type)}
+            </Text>
+            <Wallet
+              style={{
+                width: 15,
+                height: 15,
+                color: colors.n9,
+                marginLeft: 8,
+                marginBottom: 2
+              }}
+            />
+          </View>
+        </View>
+        <CellValue
+          binding={getBalanceQuery(account)}
+          type="financial"
+          style={{ fontSize: 16, color: colors.n3 }}
+          getStyle={value => value < 0 && { color: colors.r4 }}
+        />
+      </Button>
+    </View>
+  );
+}
+
+function EmptyMessage({ onAdd }) {
+  return (
+    <View style={{ flex: 1, padding: 30 }}>
+      <Text style={styles.text}>
+        For Actual to be useful, you need to add an account. You can link an
+        account to automatically download transactions, or manage it locally
+        yourself.
+      </Text>
+
+      <Button
+        primary
+        style={{ marginTop: 20, alignSelf: 'center' }}
+        onClick={() =>
+          isMobile()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏will this component be used in non-mobile version too? Currently it seems to be only used in the mobile version. So I think we can remove this `isMobile` check here.

Reply:
Good point. Removed the check.